docs/GLOSSARY.md

```diff
@@ -0,0 +1,273 @@
+# 📖 Glossary of Terms
```

Comment:
I have mixed feelings on glossaries. In many projects they are either 1) outdated 2) too verbose and explaining terms someone involved in mobile development should simply know 3) Not detailed enough so that someone who really needs to understand will need to look up more anyway or 4) not placed close enough to the actual terms so that people will look it up on the internet anyway.

In your view, how would this glossary help contributors be more effective?